# Physical robot parameters
v_max: 5.0
a_max: 2.5
j_max: 5.0
w_max: 1.8
anga_max: 6.283 # angular acceleration
